src/wizards/audience/views/setup/payment.js

```diff
@@ -2,28 +2,118 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { ToggleControl, TextareaControl } from '@wordpress/components';
 
 /**
  * Internal dependencies.
  */
-import { withWizardScreen, Wizard } from '../../../../components/src';
+import {
+	Grid,
+	Button,
+	Wizard,
+	withWizardScreen,
+} from '../../../../components/src';
 import WizardsTab from '../../../wizards-tab';
+import WizardsSection from '../../../wizards-section';
 import Platform from '../../components/platform';
 import PaymentGateways from '../../components/payment-methods';
 import NRHSettings from '../../components/nrh-settings';
 import BillingFields from '../../components/billing-fields';
 
-export default withWizardScreen( function () {
+export default withWizardScreen( function ( {
+	config,
+	getSharedProps,
+	updateConfig,
+	saveConfig,
+} ) {
```

Comment:
Suggestion: do you think it'd be too big of a refactor to move this to use the wizard store, as it's being used for 
`newspack-audience/payment`?

It allows for the component to not have to pass down all these props and the "Checkout Configuration" wizard section can become its own standalone component, like `<PaymentGateways />` and the others.

Using the store doesn't seem too different from what it's doing [here](https://github.com/Automattic/newspack-plugin/blob/13d6d55f6a3949ed17118a98369525cfc12ab442/src/wizards/audience/views/setup/index.js#L33-L66):

```js
const data = Wizard.useWizardData( 'newspack-audience/reader-activation' );

updateWizardSettings( {
	slug: 'newspack-audience/reader-activation'
} );

saveWizardSettings( {
	slug: 'newspack-audience/reader-activation'
} );
```

Reply:
@miguelpeixe thank you for the review and suggestion. See refactor(s) in 58020abef2b88a5f5430a01173616ef629c3b852, c5f5e6110d7cd68a898e4eb682718ab5577d9897, 3108cde1e004fad558d00cc6216d52ee3382bb2f, 0f6f5cef993f83e1792fde4b84c827a289155a23

Reply:
Thank you for the refactor, @jaredrethman! It works great.